contrib/jenkins/init_cluster.sh

```diff
@@ -67,5 +69,12 @@ echo "Using cluster ${CLUSTERNAME}."
 gcloud container clusters get-credentials "${CLUSTERNAME}" --project="${PROJECT}" --zone="${ZONE}" \
   || { echo 'Cannot get credentials for cluster.'; exit 1; }
 
+# On GKE you need to give your user proper permissions in order to create new
+# cluster roles. Needed for RBAC setup.
+ACCOUNT_NAME="$(gcloud info | grep Account | sed 's/.*\[\(.*\)\]/\1/')"
+kubectl create clusterrolebinding jenkins-cluster-admin-binding \
+    --clusterrole=cluster-admin --user="${ACCOUNT_NAME}" \
```

Comment:
@arschles @kibbles-n-bytes  for default minikube I did
`kubectl create clusterrolebinding minikube-cluster-admin-binding --clusterrole=cluster-admin --user=minikube`